app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -1,13 +1,20 @@
 package com.github.fribourgsdp.radio
 
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
 import android.view.KeyEvent
 import android.view.View
 import android.view.inputmethod.EditorInfo
 import android.widget.*
-import androidx.appcompat.app.AppCompatActivity
+import io.agora.rtc.Constants
+import io.agora.rtc.RtcEngine
 import kotlinx.serialization.decodeFromString
 import kotlinx.serialization.json.Json
+import kotlin.random.Random
+
+const val IS_IN_TEST_MODE = "Test mode for voice chat engine"
```

Comment:
Linking to previous comment, is it still needed?

Reply:
No it's an oversight thanks for pointing it out.